cmd/gossamer/configcmd_test.go

```diff
@@ -141,8 +141,8 @@ func TestGetConfig(t *testing.T) {
 	cfgClone.Global.DataDir, err = filepath.Abs(cfgClone.Global.DataDir)
 	require.Nil(t, err)
 
-	app := cli.NewApp()
-	app.Writer = ioutil.Discard
+	thisapp := cli.NewApp()
```

Comment:
I don't see where the previous declaration was?

Reply:
Its:
`cmd/gossamer/main.go:29` `app       = cli.NewApp()`